index.html

```diff
@@ -408,8 +408,9 @@ <h4>DataIntegrityProof</h4>
 `controller`.
           </p>
           <p>
-The `proofValue` property of the proof MUST be an ECDSA or deterministic ECDSA
-signature produced according to [[FIPS-186-5]] using the curves and hashes as
+The `proofValue` property of the proof MUST be an ECDSA signature, in general,
+and SHOULD be a <em>deterministic</em> ECDSA signature, specific variant,
+produced according to [[FIPS-186-5]] using the curves and hashes as
```

Comment:
`specific variant` doesn't seem to make sense here. Maybe there's a rephrasing that would reintroduce it, but I don't understand what it's trying to do, so I can't offer that rephrasing.
```suggestion
and SHOULD be a <em>deterministic</em> ECDSA signature
produced according to [[FIPS-186-5]] using the curves and hashes as
```

Reply:
So the `specific variant` *is* the `deterministic` version of an ECDSA signature. There are two variants: deterministic and non-deterministic.

The text is trying to say that you MUST use an ECDSA signature (in general, *any* / *either* variant of an ECDSA signature) *and* that you SHOULD use the deterministic variant (but you can also use the non-deterministic variant if you have good reason to).

Reply:
Yes, @dlongley. I was trying to use wording aligned with FIPS-186-5. @TallTed this is some of the wording from FIPS-186-5. 

> 6.3.2 Generation of the Per-Message Secret Number for Deterministic ECDSA
> Deterministic ECDSA is a variant of ECDSA where the per-message secret number is a function of the message that is signed and the private key, thereby resulting in a deterministic mapping of messages to signatures. This protects against attacks arising from generating signatures with insufficient randomness in the per-message secret number that would reveal a private key. Deterministic ECDSA may be desirable for devices that do not have a good source of quality random numbers for generating the per-message secret number.

Reply:
> So the specific variant is the deterministic version of an ECDSA signature

Ah! I looked to the references, but couldn't find anything this clear.

```suggestion
and SHOULD be a <em>deterministic</em> ECDSA signature, also known
as <em>the specific variant of a signature</em>,
produced according to [[FIPS-186-5]] using the curves and hashes as
```

Reply:
or maybe
```suggestion
and SHOULD be a <em>deterministic</em> (or <em>specific variant</em>) ECDSA
signature, produced according to [[FIPS-186-5]] using the curves and hashes as
```